portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,11 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+async function loadingcontent() {
```

Comment:
See https://google.github.io/styleguide/jsguide.html#naming-method-names, method names should be written in lowerCamelCase like "lowerCamelCase"

Reply:
Fixed